appservice/src/createAppService/AppInsightsCreateStep.ts

```diff
@@ -54,6 +62,18 @@ export class AppInsightsCreateStep extends AzureWizardExecuteStep<IAppServiceWiz
         }
     }
 
+    public async selectExistingPrompt(wizardContext: IAppServiceWizardContext): Promise<void> {
+        const message: string = localize('planForbidden', 'You do not have permission to create an app insights resource in subscription "{0}".', wizardContext.subscriptionDisplayName);
+        const selectExisting: MessageItem = { title: localize('selectExisting', 'Select Existing') };
+        wizardContext.telemetry.properties.cancelStep = 'AppInsightsNoPermissions';
+        await wizardContext.ui.showWarningMessage(message, { modal: true }, selectExisting);
```

Comment:
It feels like this prompt should have the "Skip for Now" option because it's not obvious that "Select Existing" will eventually let me skip

Reply:
Do you mean have "Skip for Now" directly in the warning along with "Select Existing"?

Reply:
Something like this...? 
![image](https://user-images.githubusercontent.com/17537419/118879167-3ac47400-b8a5-11eb-8686-88c657878e75.png)

Reply:
Yes exactly